internal/mempool/mempool_test.go

```diff
@@ -2285,3 +2313,239 @@ func TestRemoveDoubleSpends(t *testing.T) {
 			"exist in pool.", ticket.Hash())
 	}
 }
+
+// TestHandlesTSpends verifies that the mempool correctly adds and removes
+// valid tspends and limits their total number according to the appropriate
+// limits.
+func TestHandlesTSpends(t *testing.T) {
+	t.Parallel()
+
+	// TODO: switch to mainnet params once they have treasury params set so
```

Comment:
The parameters (with the exception of the Pi key which can't be used here anyway as previously noted) have been added in #2170, so this can be updated accordingly.